v_4_9_1/release.go

```diff
@@ -0,0 +1,62 @@
+package v_4_9_1
+
+import (
+	"fmt"
+
+	"github.com/giantswarm/apiextensions/pkg/apis/release/v1alpha1"
+)
+
+func BuildImages(registryDomain string, versions Versions) Images {
+	return Images{
+		CalicoCNI:             buildImage(registryDomain, "giantswarm/cni", versions.Calico),
```

Comment:
These are just for defaulting. The provider operator will call this, but it's free to override any image before sending them to the templating step.